pydantic/fields.py

```diff
@@ -960,6 +1081,28 @@ def __eq__(self, other: Any) -> bool:
         )
 
 
+_T = TypeVar('_T')
```

Comment:
Do we really need to define this twice?

Reply:
Ah did not notice they were defined in the same file